### ISSUE TITLE
Development 

### ISSUE THREAD

**Issue by Lucifer0601code** [astarivi/Kaizoyu#15]
This app was a very good and had pleasant looking UI and UX. i will support the development team if anything they could do to bring it back.

**Comment by astarivi** [comment]
I'm glad to hear you enjoyed the app, it was a passion project of mine for some time, and I dedicated significant effort to delivering a great user experience. However, continued development requires substantial time and resources. 

At the time, I chose not to monetize the app, which, among other reasons, lead to a pause in development as further milestones leading to the 1.0 release were getting increasingly difficult to reach. Nevertheless, I have since change my mind about monetization, and if there’s interest in providing financial support, I would be open to resuming development to achieve the 1.0 release.

Main milestones for 1.0 are:
- [Fix the search enhancer by possibly turning Kaizoyu into a pseudo "social network"](https://github.com/users/astarivi/projects/1?pane=issue&itemId=84227084). This issue replaces past milestone [In-app missing show report system](https://github.com/users/astarivi/projects/1?pane=issue&itemId=32223230), and is related to #3 
- Work on #1
- Update the anime downloading component (and potentially add torrent support?)
- Update the app to latest Android version
- Replace Kitsu with AniList (optional)

For more details, check the [development board](https://github.com/users/astarivi/projects/1/views/1)
